core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/UnifyingNumbers.kt

```diff
@@ -12,16 +14,23 @@ package org.jetbrains.kotlinx.dataframe.documentation
  * For each number type in the graph, it holds that a number of that type can be expressed lossless by
  * a number of a more complex type (any of its parents).
  * This is either because the more complex type has a larger range or higher precision (in terms of bits).
+ *
+ * There are variants of this graph that exclude some types, such as `BigDecimal` and `BigInteger`.
+ * In these cases `Double` could be considered the most complex type.
+ * `Long`/`ULong` and `Double` could be joined to `Double`,
+ * potentially losing a little precision, but a warning will be given.
+ *
+ * See [UnifiedNumberTypeOptions] for these settings.
  */
 internal interface UnifyingNumbers {
 
     /**
      * ```
-     *            BigDecimal
+     *           (BigDecimal)
```

Comment:
Will it be removed in the next PRs and will you keep the DAG after removal?

Reply:
JSON can still create Big numbers and maybe other places where number unification needs to happen can do it. It's only in statistics where we don't want big numbers, right?